### PR TITLE
DEV-4922: hacker-news fix audit and readme

### PR DESCRIPTION
**Pull request by MSilva95**
fix audit and readme

### CODE REVIEW

File: subjects/mobile-dev/hacker-news/README.md [hunk 1]
```diff
@@ -1,36 +1,31 @@
...
+Your app should have the following functionalities:
 
-### Objective
+- Login button. Once you tap it a log in page must be displayed.
+- Show a list of all fetched `posts` on the main screen.
+- Each `post` should have an `upvote` option for voting.
+- Each `post` should display the total number of votes and comments they have.
+- Display the `username` of the `post` creator.
+- Display the submission time of the `post`.
+- Open the link in a [Webview](https://codelabs.developers.google.com/codelabs/flutter-webview#0) when a `post` is tapped.
```
Comment (nprimo) [inline subjects/mobile-dev/hacker-news/README.md:15]: Little detail - in the link `WebView` is the format used for the word

File: subjects/mobile-dev/hacker-news/audit/README.md [hunk 2]
```diff
@@ -1,17 +1,29 @@
 #### Functional
 
-> In order to run and hot reload app either on emulator or device, follow the [instructions](https://docs.flutter.dev/get-started/test-drive?tab=androidstudio#run-the-app)
+###### Does the app show a list of all fetched posts on the main screen?
 
-###### Does app shows all fetched posts?
+###### When you tap on a post, does it open the content in a `webview`?
```
Comment (nprimo) [inline subjects/mobile-dev/hacker-news/audit/README.md:5]: Little detail - in the link `WebView` is the format used for the word

File: subjects/mobile-dev/hacker-news/audit/README.md [hunk 3]
```diff
@@ -1,17 +1,29 @@
...
 
-###### Tap any post, does it open webview of the content?
+###### Is it possible to go back to the main page from the `webview`?
```
Comment (nprimo) [inline subjects/mobile-dev/hacker-news/audit/README.md:7]: Little detail - in the link `WebView` is the format used for the word

**Review by nprimo — APPROVED** [review]
LGTM! I have added just a small comment about the format of a key word

File: subjects/mobile-dev/hacker-news/README.md (hunk 1 above)
Comment (MSilva95) [reply subjects/mobile-dev/hacker-news/README.md]: done

File: subjects/mobile-dev/hacker-news/audit/README.md (hunk 2 above)
Comment (MSilva95) [reply subjects/mobile-dev/hacker-news/audit/README.md]: done

File: subjects/mobile-dev/hacker-news/audit/README.md (hunk 3 above)
Comment (MSilva95) [reply subjects/mobile-dev/hacker-news/audit/README.md]: done

**Pull request merged**